dlf/plugins/toc/flexform.xml

```diff
@@ -54,7 +54,7 @@
 								<allowed>pages</allowed>
 								<size>1</size>
 								<maxitems>1</maxitems>
-								<minitems>1</minitems>
+								<minitems>0</minitems>
```

Comment:
Why did you change this? How does the TOC plugin make any sense without a link target?

Reply:
I am still interested in your intentions behind this change. ;)